src/main/kotlin/racingCar/strategy/MoveStrategy.kt

```diff
@@ -0,0 +1,11 @@
+package racingCar.strategy
+
+interface MoveStrategy {
+    fun moveCar(): Int
```

Comment:
이동을 결정하는 전략이라고 한다면, boolean 값을 리턴하는게 어떨까요?

```suggestion
    fun isMove(): Boolean
```

현재는 `moveCar`에서 Int 값을 받은 뒤 아래 메소드에 다시 Int 값을 넘겨주고 있습니다.

```kotlin
    fun changeStateRacingCar(inputRandomNum: Int) {
        if (inputRandomNum >= STANDARD_VALUE_TO_MOVE) {
            return move()
        }
    }
```

그리고 내부에서 또 if문으로 체크를 진행하고 있어요, Strategy에서 바로 boolean으로 이동 여부를 리턴하면 어떨까요?